packages/core-entities/src/models/views/YieldViews.ts

```diff
@@ -41,6 +39,10 @@ export type ProductGroupItem = {
   underlying?: TokenDefinition;
   collateralFactor: string;
   debtToken?: TokenDefinition;
+  vaultData?: {
```

Comment:
Why does this need to be nested like this? The debtToken is returned twice and what is the purpose of returning the non-leveraged APY?